showcase.py

```diff
@@ -0,0 +1,81 @@
+from pynamodb.attributes import (
+    UnicodeAttribute, UnicodeSetAttribute, Attribute, LegacyBooleanAttribute,
+    BooleanAttribute, NumberAttribute, BinaryAttribute,MapAttribute, ListAttribute, EnumAttribute)
+
+from pynamodb.models import Model
+
+class MyModel(Model):
+    class Meta:
+        table_name = "TestModel"
+        host="http://localhost:8000"
+    id = UnicodeAttribute(hash_key=True)
+    enum_attr = EnumAttribute(type_set={UnicodeAttribute(null=True), NumberAttribute(null=True), UnicodeSetAttribute(null=True)}, null=True)
+
+if MyModel.exists():
+    MyModel.delete_table()
+MyModel.create_table(read_capacity_units=2, write_capacity_units=2)
+
+
+############# Basic Usage ###############
+# Create an object
+m1 = MyModel("id1")
+m1.enum_attr['UnicodeAttribute'] = "Obama"
```

Comment:
Could we end up with a . notation instead of the indexing notation?

Like m1.enum_attr.UnicodeAttribute = "Obama"

Or even better potentially infer it based on type?

Reply:
I am considering the dot notion, just need to get them to work...

Reply:
:)